<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleWindow</class>
 <widget class="QWidget" name="BattleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Battle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resfile.qrc">
    <normaloff>:/res/images/boss.png</normaloff>:/res/images/boss.png</iconset>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/images/battleImg.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>42</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 140);
color: rgb(255, 255, 255);
font: 14pt &quot;MS UI Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="oponentLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 140);
color: rgb(255, 255, 255);
font: 14pt &quot;MS UI Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 140);
color: rgb(255, 255, 255);
font: 14pt &quot;MS UI Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="attackButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="escapeButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Escape</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inventoryButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resfile.qrc"/>
 </resources>
 <connections/>
</ui>
